src/test/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtilsUnitTest.java

```diff
@@ -38,28 +36,42 @@ public void testExecuteQueryWithWhereClause() {
         expectedNamesAndAges.put("Fred", "35");
 
         final String query = String.format("SELECT * FROM `%s` WHERE name = 'Fred'", BIGQUERY_FULLY_QUALIFIED_TABLE);
+        Map<String, String> labels = new HashMap<String, String>();
+        labels.put("gatktestquery", "testwhereclause" + runUuid);
+        System.out.print("testwhereclause" + runUuid);
```

Comment:
Use the logger, not `System.out.print()`

Reply:
I think they're more commonly called lumberjacks?  
![image](https://user-images.githubusercontent.com/4700332/231017608-74ced2d6-1995-45aa-ab32-0ccd9e71fb7b.png)